src/core/cardano/addresses.ts

```diff
@@ -1,27 +0,0 @@
-import { Bip32PrivateKey } from "@dcspark/cardano-multiplatform-lib-browser";
```

Comment:
Please check the story - this dependency should be removed from our package json files. We should also remove references to it in `jest.config.ts`